Source/WebCore/html/canvas/WebGLRenderingContextBase.h

```diff
@@ -625,9 +628,10 @@ class WebGLRenderingContextBase : public GraphicsContextGL::Client, public GPUBa
     GCGLint m_maxTextureSize;
     GCGLint m_maxCubeMapTextureSize;
     GCGLint m_maxRenderbufferSize;
-    GCGLint m_maxViewportDims[2] { 0, 0 };
+    std::array<GCGLint, 2> m_maxViewportDims { 0, 0 };
```

Comment:
What does switching from `GCLint[2]` to `std::array<GCGLint, 2>` provide?

Reply:
assignment in the context initialization.
The assignment is needed, since the array getter might fail and not write anything, leaving the value of the array as the previous value